apps/src/applab/designProperties.jsx

```diff
@@ -48,6 +29,7 @@ var DesignProperties = module.exports = React.createClass({
     // TODO (brent) - right now if i create two elements with the same id, I
     // can still run into the same problem, where I click on the other element
     // and the table doesn't update
+    // TODO (brent) - it appears the wrong element sometimes gets deleted
     return (
       <div key={this.props.element.id}>
```

Comment:
Better solution might be to come up with a different source of our key. One thought I just had was ordering within the parent div. Of course then we'll have to consider what happens as items get deleted, or as we move this around one we have move forward/move back, etc.